### PR TITLE
Feature/db select

### PR DESCRIPTION
**Pull request by ayafuji**
- close with await
- add db variable to select db

### CODE REVIEW

File: trio_redis/client.py [hunk 1]
```diff
@@ -16,8 +16,8 @@ class Redis:
       b'42'
     """
 
-    def __init__(self, addr=b"127.0.0.1", port=6379):
-        self.conn = RedisConnection(addr, port)
+    def __init__(self, addr=b"127.0.0.1", port=6379, db=None):
```
Comment (belm0) [inline trio_redis/client.py]: default db should be 0, like redis-cli.  Same for RedisConnection.

File: trio_redis/connection.py [hunk 2]
```diff
@@ -30,21 +30,24 @@ class RedisConnection:
...
+            await self.process_command_ok(b"SELECT", self.db)
 
     def close(self):
         self.sock.close()
 
     async def send_command(self, command, *args):
         command_and_args = (serialize(arg) for arg in (atom(command),) + args)
         data = b" ".join(command_and_args) + b"\r\n"
-        await self.sock.sendall(data)
+        await self.sock.send(data)
```
Comment (belm0) [inline trio_redis/connection.py]: I'd prefer if trio upgrade was done as separate PR.  It should include change to requirements.txt

Comment (ayafuji) [reply trio_redis/connection.py]: Sorry I could not get the point. you mean "specify trio version in requirement.txt" ?

Comment (belm0) [reply trio_redis/connection.py]: yes, because current version in requirements.txt is old (trio~=0.1)

Comment (ayafuji) [reply trio_redis/connection.py]: I see. I updated requirement.txt !

File: requirements.txt [hunk 3]
```diff
@@ -1 +1 @@
-trio~=0.1
+trio~=0.11.0
```
Comment (belm0) [inline requirements.txt]: maybe `>=0.11.0`

neodm is already using 0.12

File: trio_redis/connection.py [hunk 4]
```diff
@@ -30,21 +30,24 @@ class RedisConnection:
...
-    def __init__(self, addr, port, bufsize=16384):
+    def __init__(self, addr, port, db, bufsize=16384):
         self.addr = (addr, port)
+        self.db = db
         self.sock = trio.socket.socket()
         self.bufsize = bufsize
 
     async def connect(self):
         await self.sock.connect(self.addr)
+        if self.db :
```
Comment (belm0) [inline trio_redis/connection.py]: I think it can always be sent
